swift/Workflow/Sources/WorkflowHost.swift

```diff
@@ -53,6 +53,19 @@ public final class WorkflowHost<WorkflowType: Workflow> {
 
     }
 
+    /// Update the input for the workflow. Will cause a render pass.
+    public func update(workflow: WorkflowType) {
+        rootNode.update(workflow: workflow)
+
+        // Treat the update as an "output" from the workflow as an external event to force a render pass.
```

Comment:
Does this mean that the host emits an output every time the root workflow "input" is updated?

Reply:
At the `WorkflowHost` level, it "outputs" every time something in the tree changes. Sometimes that's an actual output with the `Output` type on the workflow, otherwise it's an event that includes the debugging information and is the signal that a render pass needs to occur.